jetty9/src/main/java/com/thoughtworks/go/server/Jetty9Server.java

```diff
@@ -76,7 +76,10 @@ public Jetty9Server(SystemEnvironment systemEnvironment, String password) {
     public void configure() throws Exception {
         server.addEventListener(mbeans());
         server.addConnector(plainConnector());
-        server.addConnector(sslConnector());
+        Connector connector = sslConnector();
+        if (connector != null) {
+            server.addConnector(connector);
+        }
```

Comment:
@arvindsv and @maheshp is this OK for an implementation? Basically don't configure ssl on fresh installs.

Reply:
@ketan My opinion: I'm happy to replace these kinds of things with two pieces of documentation / help:

1. Upon an upgrade, what do I need to do to keep everything the way it was? [Probably nothing in this case?]

2. What do I need to do and in what order to get a new installation to use TLS termination between the agents and server, if I want.

3. Optional: Warning users that it is not secure to run in an untrusted/open network, without doing what point 2 says.

As long as that is clear, I think it's fine.